### PR TITLE
lint the whole repo, fix a bug found by pyflakes

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
:+1:

**Comment by bwrsandman** [comment]
:-1: !!

Do not put noqa so casually! This is bad practice.

@pedrobaeza 3h to review a merge is not OK! Especially when you're the only member of the maintainer team to do a review!

File: account_analytic_line_list/__init__.py [hunk 1]
```diff
@@ -19,4 +19,4 @@
 #
 ##############################################################################
 
-from . import wizard
+from . import wizard  # noqa
```
Comment (bwrsandman) [inline account_analytic_line_list/__init__.py:22]: This is not necessary and is better without.

File: account_analytic_line_list/wizard/__init__.py [hunk 2]
```diff
@@ -21,4 +21,4 @@
 #
 ##############################################################################
 
-from . import account_analytic_line
+from . import account_analytic_line  # noqa
```
Comment (bwrsandman) [inline account_analytic_line_list/wizard/__init__.py:24]: This is not necessary and is better without.

File: account_analytic_required/__init__.py [hunk 3]
```diff
@@ -21,4 +21,4 @@
 ##############################################################################
 
 
-from . import account
+from . import account  # noqa
```
Comment (bwrsandman) [inline account_analytic_required/__init__.py:24]: This is not necessary and is better without.

File: analytic_multicurrency/__init__.py [hunk 4]
```diff
@@ -19,6 +19,6 @@
 #
 ##############################################################################
 
-import account_analytic_line
-import analytic
-import analytic_analysis
+import account_analytic_line  # noqa
+import analytic  # noqa
+import analytic_analysis  # noqa
```
Comment (bwrsandman) [inline analytic_multicurrency/__init__.py:24]: `noqa` is not necessary and is better without.

File: analytic_secondaxis/__init__.py [hunk 5]
```diff
@@ -29,5 +29,5 @@
 #
 ##############################################################################
 
-import analytic_secondaxis
-import wizard
\ No newline at end of file
+import analytic_secondaxis  # noqa
+import wizard  # noqa
```
Comment (bwrsandman) [inline analytic_secondaxis/__init__.py:33]: `noqa` is not necessary and is better without.

File: analytic_secondaxis/wizard/__init__.py [hunk 6]
```diff
@@ -29,4 +29,4 @@
 #
 ##############################################################################
 
-import analytic_activity_chart
+import analytic_activity_chart  # noqa
```
Comment (bwrsandman) [inline analytic_secondaxis/wizard/__init__.py:32]: `noqa` is not necessary and is better without.

File: analytic_secondaxis_multicurrency/__init__.py [hunk 7]
```diff
@@ -29,4 +29,4 @@
 #
 ##############################################################################
 
-import analytic_secondaxis
+import analytic_secondaxis  # noqa
```
Comment (bwrsandman) [inline analytic_secondaxis_multicurrency/__init__.py:32]: `noqa` is not necessary and is better without.

File: analytic_secondaxis_project/__init__.py [hunk 8]
```diff
@@ -29,4 +29,4 @@
 #
 ##############################################################################
 
-import analytic_secondaxis
+import analytic_secondaxis  # noqa
```
Comment (bwrsandman) [inline analytic_secondaxis_project/__init__.py:32]: `noqa` is not necessary and is better without.

File: hr_expense_analytic_plans/__init__.py [hunk 9]
```diff
@@ -18,4 +18,4 @@
 #
 ##############################################################################
 
-from . import model
+from . import model  # noqa
```
Comment (bwrsandman) [inline hr_expense_analytic_plans/__init__.py:21]: `noqa` is not necessary and is better without.

File: hr_expense_analytic_plans/model/__init__.py [hunk 10]
```diff
@@ -20,4 +20,4 @@
 #
 ##############################################################################
 
-from . import hr_expense_analytic_plans
+from . import hr_expense_analytic_plans  # noqa
```
Comment (bwrsandman) [inline hr_expense_analytic_plans/model/__init__.py:23]: `noqa` is not necessary and is better without.

**Comment by pedrobaeza** [comment]
We talk about 2 reviews + Travis tests, but I don't see a real problem. Why is # noqa so important?

**Comment by bwrsandman** [comment]
@pedrobaeza 2 reviews of the [reviewer team](https://github.com/orgs/OCA/teams/accounting-maintainers) and another to do the merge + a 3 day wait period

or are we not following the rules of http://pad.openerp.com/p/community-review?

> Basic review processes and rules
> Please respect a few basic rules:
> 
>    At least two reviewer must approve the Merge proposal in order to be able to merge it
> 
>    A MP cannot be merged before 5 calendar days or 3 working days (based on this thread https://lists.launchpad.net/openerp-community/msg02733.html, https://lists.launchpad.net/openerp-community/msg02735.html, one say 5 calendar days and the other 3 working days, which are close, but I don't think that any final decision has been made)
> 
>    That means the MP has at least 2 approved reviews
> 
>    But the 5 calendar / 3 working days rule can be lifted if a MP is approved by at least 3 reviewers
> 
>    The person who approve a Merge proposal shouldn’t merge it himself, a third person must do it

In this case, you are the only member of the reviewer team (correct me if I am wrong @eLBati) and you did the merge yourself after 3h before anyone could get to it.

**Comment by bwrsandman** [comment]
Not only is `#noqa` a cheap way of tricking travis into not testing the line/function/module, it promotes bad practice and in the best case is a unecessary line change.
We don't add `#noqa` in other modules, why are we adding it here?

**Comment by pedrobaeza** [comment]
OK, but we can make it on other PR. I thought that this only tricks the exact line, not the rest of the file.

Regards.

**Comment by bwrsandman** [comment]
Yes best to do that.

**Comment by max3903** [comment]
Please respect a few basic rules:
- Two reviewers must approve a merge proposal in order to be able to merge it
- 5 calendar days must be given to be able to merge it
- A MP can be merged in less that 5 calendar days if and only if it is approved by 3 reviewers. If you are in a hurry just send a mail at openerp-community-reviewer@lists.launchpad.net or ask by IRC (FreeNode server, openobject channel).

https://doc.odoo.com/contribute/05_developing_modules/#community-review

**Comment by pedrobaeza** [comment]
@max3903 You don't have talk me out about community rules, because I know them perfectly, but we are in extraordinay moments where:
- people are at holidays
- there are a lot of work to do
- sometimes there are no reviewers for such these trivial things (this is not the case, but you can remember Alexandre Fayolle complaint about the absence of reviewers).
- Other people have merged with only one approvals some PRs.
- This doesn't break anything.

So let's make another PR to acommodate Sandy's complaints, and continue working.

Regards.

**Comment by max3903** [comment]
@pedrobaeza 

I think 'extraordinary moments' fits in : 
- A MP can be merged in less that 5 calendar days if and only if it is approved by 3 reviewers. If you are in a hurry just send a mail at openerp-community-reviewer@lists.launchpad.net or ask by IRC (FreeNode server, openobject channel).

**Comment by pedrobaeza** [comment]
The question is the same if nobody answer that mail, but don't worry, I will follow exactly the rules (and this is without acridity).

**Pull request merged**